force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -162,17 +213,18 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String repeatString(String inputString, Integer repeatCount) {
         // Initialize the result String
-
+        String result = '';
         // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
-            // If it is not the last iteration, add a semicolon to separate the strings
+        
+        for (Integer i = 1; i <= repeatCount; i++) {
+            result = result + inputString;
+            if (i < repeatCount) {
+                result = result + '; ';
+            } 
+            System.debug(result);
         }
-        */
-
         // Return the final result string
-        return null; // Replace null with the variable you used to store the result
+        return result; 
```

Comment:
Good work. Take a look at the String methods repeat and removeEnd(). Also, as a best practice, delete System.debug statements. You don't want those to go to Prod 

https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_string.htm